recipes/musdb/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "musdb" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - git_url: https://github.com/sigsep/sigsep-mus-db.git
+    git_tag: "v{{ version }}"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<36]
```

I think this recipe can be ``noarch``
```suggestion
  noarch: python
```

---

Fixed, resolving conversation.